lib/tapioca/compilers/dsl/active_storage.rb

```diff
@@ -0,0 +1,42 @@
+# typed: strict
+# frozen_string_literal: true
+
+begin
+  require "active_record"
+  require "active_storage"
+  require "active_storage/attached"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      class ActiveStorage < Base
+        extend T::Sig
+
+        ReflectionType = T.type_alias do
+          T.any(::ActiveStorage::Reflection::HasOneAttachedReflection,
+            ::ActiveStorage::Reflection::HasManyAttachedReflection)
+        end
+
+        sig { override.params(root: RBI::Tree, constant: T.class_of(::ActiveRecord::Base)).void }
+        def decorate(root, constant)
+          return if constant.reflections.empty?
+
+          root.create_path(constant) do |scope|
+            constant.attachment_reflections.each do |field_name, _value|
+              scope << RBI::Method.new(field_name)
+              scope << RBI::Method.new("#{field_name}=")
```

Comment:
I think you're missing the parameter `attachable` for the setter: https://github.com/rails/rails/blob/83217025a171593547d1268651b446d3533e2019/activestorage/lib/active_storage/attached/model.rb#L138